README.md

```diff
@@ -25,7 +29,9 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-No further action is required, but you may want to check out some of the available configuration options:
+No further action is required on your part. When you register a new release with the Julia General Registry (either manually or with [Registrator](https://github.com/JuliaRegistries/Registrator.jl/blob/master/README.md)), TagBot will create a Github Tag and Github Release on your package's Github repository.  
```

Comment:
```suggestion
No further action is required on your part.
When you add a new release to a registry with Registrator, TagBot will create a GitHub release on your package's repository.
```

Just being a bit less verbose:
- Releases aren't necessarily in General
- I don't think anyone is manually editing the registry
- We already linked to Registrator above
- Most of the time, TagBot will create lightweight tags so just mentioning that it creates a GitHub release is sort of more accurate